### DISCUSSION TITLE
Difference between this and turborepo's documented caching strategy

### DISCUSSION THREAD

**Discussion by rharkor** [rharkor/caching-for-turbo#54] [General]
Original message from @dhyeyyyy

Hey, was just curious what the difference is between this action and the steps provided in turbo's [docs](https://turbo.build/repo/docs/guides/ci-vendors/github-actions#caching-with-github-actionscache)? Thanks!!

**Comment by rharkor** [comment]
Hi @dhyeyyyy,

I think both solutions work great, the main difference is that caching with the GitHub action cache will cache the whole turbo output and download it each time, for example if you have 5 different apps it will store the cache of these 5 apps and download it every time you use the cache action.

I prefer simulating a distant "remote caching server" as it may be more configurable and powerful, because it will download only the file needed on build instead of the whole turbo directory.

TLDR;
Both are great but this solution ("remote caching server") may be powerful for bigger monorepo 😉

**Reply by niklas-paro** [reply]
@rharkor If you want more users I would suggest you add this to the main Readme, as it seems like the primary selling point of the action.

**Reply by rharkor** [reply]
Hey, you are right I will definitely do that, thank you!
